sunpy/map/maphelper.py

```diff
@@ -0,0 +1,200 @@
+from sunpy import map
+from sunpy.util import MetaDict
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from sunpy.sun import constants as con
+from sunpy.time import parse_time
+import numpy as np
+import random
+import matplotlib.pyplot as plt
+import astropy
+from sunpy.coordinates import frames
+import warnings
+from sunpy.util.exceptions import SunpyUserWarning
+
+__all__ = ['meta_keywords', 'make_header']
+
+
+def meta_keywords(*key):
+    """
+    Prints the accepted metadata keywords that are used when
```

Comment:
yes so accepted meta keywords here means essentially what keywords are actually taken from the meta and used in the generation of the map, of course more can be given they just wont necessarily be called in `GenericMap` (for example #1832). I think this function is helpful so that the user can see what the actual keywords are being used - but it may be better suited on the docs? Or if there was a way we could scrape from `mapbase.py` so that it gets updated

Reply:
> I think this function is helpful so that the user can see what the actual keywords are being used

I can see the utility here, and I think we should keep it, I do think we need a more appropriate word than "accepted" though.

> Or if there was a way we could scrape from mapbase.py so that it gets updated.

Not easily enough that it's worth doing in this PR, I have an idea or two about how we could go about doing this if we wanted to. I don't think this set of data is likely to change very often so I am happy to keep it as it is.

If this is just for reference then maybe we should just be auto-generating some documentation rather than providing it as a function?